.github/workflows/nightly.yml

```diff
@@ -0,0 +1,25 @@
+name: Nightly
```

Comment:
Since the only thing this does is check for links, I'd rather see it named "Check Links".